plio/tests.py

```diff
@@ -439,6 +439,56 @@ def test_org_user_can_list_other_user_items(self):
         # set db connection back to public (default) schema
         connection.set_schema_to_public()
 
+    def test_guest_cannot_update_items(self):
+        # unset the credentials
+        self.client.credentials()
+        # update item
+        response = self.client.put(
+            f"/api/v1/items/{self.item.id}/", {"time": 2, "plio": self.plio.id}
+        )
+        self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+
+    def test_user_can_update_own_items(self):
+        # update item
+        response = self.client.put(
+            f"/api/v1/items/{self.item.id}/", {"time": 2, "plio": self.plio.id}
+        )
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
along with this we can also assert that the item was actually updated? Just to be double sure!